### PR TITLE
Add metadata to Kiln transactions

### PR DESCRIPTION
**Pull request by peachbits**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [ ] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [ ] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208642530820111

### CODE REVIEW

File: src/plugins/stake-plugins/generic/policyAdapters/EthereumKilnAdaptor.ts [hunk 1]
```diff
@@ -33,6 +33,11 @@ export const makeEthereumKilnAdapter = (policyConfig: StakePolicyConfig<Ethereum
   const provider = new ethers.providers.FallbackProvider(rpcProviderUrls.map(url => new ethers.providers.JsonRpcProvider(url)))
   const integrationContract = KilnLiquid20A__factory.connect(contractAddress, provider)
 
+  // Metadata constants:
+  const metadataName = 'Kiln Pooled Staking'
```
Comment (swansontec) [inline src/plugins/stake-plugins/generic/policyAdapters/EthereumKilnAdaptor.ts:37]: Do we usually localize metadata strings?

File: src/plugins/stake-plugins/generic/policyAdapters/EthereumKilnAdaptor.ts [hunk 2]
```diff
@@ -33,6 +33,11 @@ export const makeEthereumKilnAdapter = (policyConfig: StakePolicyConfig<Ethereum
...
+  const stakeAsset = policyConfig.stakeAssets[0]
+  const metadataPoolAssetName = `${stakeAsset.currencyCode}`
```
Comment (swansontec) [inline src/plugins/stake-plugins/generic/policyAdapters/EthereumKilnAdaptor.ts:39]: Is the `'${}'` redundant?

**Review by swansontec — APPROVED** [review]

File: src/plugins/stake-plugins/generic/policyAdapters/EthereumKilnAdaptor.ts (hunk 1 above)
Comment (peachbits) [reply src/plugins/stake-plugins/generic/policyAdapters/EthereumKilnAdaptor.ts:37]: It's a product name so we can leave it untranslated

File: src/plugins/stake-plugins/generic/policyAdapters/EthereumKilnAdaptor.ts (hunk 2 above)
Comment (peachbits) [reply src/plugins/stake-plugins/generic/policyAdapters/EthereumKilnAdaptor.ts]: yup, fixed

**Pull request merged**